src/commit/account.rs

```diff
@@ -438,17 +438,26 @@ impl<A: AccountPatch> AccountState<A> {
     /// Find code by its address in this account state. If the search
     /// failed, returns a `RequireError`.
     pub fn code(&self, address: Address) -> Result<Rc<Vec<u8>>, RequireError> {
+        let code = self.code_opt_nonexist(address)?
```

Comment:
"?.unwrap_or_else()" - not a best practice, 
better to use "?" just on the end of expression, and use explicit methods inside chained calls.

Reply:
Agreed. Fixed that.